persistence/shared_sql.go

```diff
@@ -83,6 +86,19 @@ func (p *PostgresContext) GetActorsUpdated(actorSchema types.ProtocolActorSchema
 	return
 }
 
+func (p *PostgresContext) getAccountAmount(accountSchema types.ProtocolAccountSchema, address string, height int64) (amount string, err error) {
```

Comment:
Can the functions here go into a separate `account_shared_sql`?

Might mean we have to rename this to `actor_shared_sql`